modules/cli/src/main/scala/scala/cli/commands/Test.scala

```diff
@@ -145,14 +145,17 @@ object Test extends ScalaCommand[TestOptions] {
         val linkerConfig = build.options.scalaJsOptions.linkerConfig(logger)
         value {
           Run.withLinkedJs(build, None, addTestInitializer = true, linkerConfig, logger) { js =>
-            Runner.testJs(
+            val res = Runner.testJs(
               build.fullClassPath,
               js.toIO,
               requireTests,
               args,
               testFrameworkOpt,
               logger
             )
+            // clean after linking js TODO it should be move to Run.withLinkedJs
```

Comment:
We shouldn't call `withLinkedJs` here. `withLinkedJs`'s only purpose is cleaning-up that file. We can just call `Package.linkJs` ourselves here.